canonical-websites/www.ubuntu.com
# Move and update the OpenStack page

## Done
Updated the content with the latest copy doc. Moved from `/cloud/openstack` to `openstack`. Updated all links to it across the site.

Removed the `/cloud/openstack/autopilot` page and removed all links to it.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check the copy matches the [copy doc](https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit)
- Check the menu is correct

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2186
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2221
